[ResponseOps][Reporting] Scheduled report flyout UI

umbopepato opened the pull request:
## Summary

> [!IMPORTANT]
> This PR is targeting the `scheduled-reports-ui` feature branch, where the backend changes from the `scheduled-reports` branch are temporarily integrated while waiting for https://github.com/elastic/kibana/pull/221028 to be merged (see the squashed `[TMP] ...` commit message).

Implements the flyout UI for the creation and read-only viewing of scheduled reports (exports).

<img height="500" alt="image" src="https://github.com/user-attachments/assets/6bbd274b-1bbb-481f-ac40-4d7131be6d85" />

<details>

<summary>

## Known issues

</summary>

- Console error due to `compressed` attribute wrongly forwarded by form-hook-lib to DOM element (this is likely a form lib issue):
    <img width="916" alt="image" src="https://github.com/user-attachments/assets/09d20ba9-8781-46d6-bcfa-862d8a4cbf90" />

- Email validation errors accumulate instead of replacing the previous one (again looks like a fom lib issue):
    https://github.com/user-attachments/assets/f2dc7a46-a3a9-465d-b8a1-3187b200f9b9

</details>

<details>

<summary>

## Screenshots

</summary>

Health API error:
<img height="500" alt="Screenshot 2025-05-31 at 10 48 40" src="https://github.com/user-attachments/assets/dd069597-971c-489f-9c07-eb5edfd7bede" />

Health API loading state:
<img height="500" alt="Screenshot 2025-05-31 at 10 49 04" src="https://github.com/user-attachments/assets/27d95bf3-bf7d-42c7-9a40-2826f38aa837" />

Health API success with some missing prerequisites:
<img width="449" alt="Screenshot 2025-06-17 at 16 59 57" src="https://github.com/user-attachments/assets/c44afa97-70ff-4618-8b73-41b816514459" />

Form validation:
<img height="500" alt="image" src="https://github.com/user-attachments/assets/a8d4cae1-2819-4f71-a911-9300a6cf81f8" />

Success toast:
<img width="480" alt="image" src="https://github.com/user-attachments/assets/a87c3af5-dbb0-40e8-915a-fc9d7e1d97f2" />

Failure toast:
<img width="518" alt="image" src="https://github.com/user-attachments/assets/908f9dea-b5cb-4da9-b4a5-76e313837f18" />

Print format toggle:
<img width="502" alt="image" src="https://github.com/user-attachments/assets/602f3ab9-07ef-4689-a305-dc1b2b5495cd" />

Missing notifications email connector callout:
<img width="499" alt="image" src="https://github.com/user-attachments/assets/fe4997a5-75e6-4450-85e5-7d853049e085" />

</details>

## References

Closes #216321
Stacked on #220535

umbopepato committed:
Draft scheduled report flyout UI

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/303902)
* Commit: 910fe09a0c460c98e68a936f9284b03cb89a08c3

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/303902#0197259a-5aa6-4e36-8e20-10a60a89ce80)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303902","number":303902,"commit":"910fe09a0c460c98e68a936f9284b03cb89a08c3"}],"number":303902}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/303913)
* Commit: 4dd68c2f3692ed1c0bd5b9ae70e3a1cafab12d22

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/303913#0197275b-e33a-4db3-9851-1647b035ee60)

### History
* :broken_heart: [Build #303902](https://buildkite.com/elastic/kibana-pull-request/builds/303902) failed 910fe09a0c460c98e68a936f9284b03cb89a08c3

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303913","number":303913,"commit":"4dd68c2f3692ed1c0bd5b9ae70e3a1cafab12d22"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303902","number":303902,"commit":"910fe09a0c460c98e68a936f9284b03cb89a08c3"}],"number":303913}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/303914)
* Commit: 0d3db10a6c4421a1bdf694b06ff55f172dbc3f51

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/303914#019727b9-299f-4938-b8de-9664f8d480cd)

### History
* :broken_heart: [Build #303913](https://buildkite.com/elastic/kibana-pull-request/builds/303913) failed 4dd68c2f3692ed1c0bd5b9ae70e3a1cafab12d22
* :broken_heart: [Build #303902](https://buildkite.com/elastic/kibana-pull-request/builds/303902) failed 910fe09a0c460c98e68a936f9284b03cb89a08c3

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303914","number":303914,"commit":"0d3db10a6c4421a1bdf694b06ff55f172dbc3f51"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303913","number":303913,"commit":"4dd68c2f3692ed1c0bd5b9ae70e3a1cafab12d22"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303902","number":303902,"commit":"910fe09a0c460c98e68a936f9284b03cb89a08c3"}],"number":303914}
buildkite-pr-comment-->

umbopepato committed:
Implement scheduled reports flyout

umbopepato committed:
Fix type errors

umbopepato committed:
Fix translations, reduce page load bundle size

umbopepato committed:
Switch back to form in flyout content, add tests

elasticmachine commented:
Pinging @elastic/response-ops (Team:ResponseOps)

ymao1 reviewed (commented):
Comments from verification:

**License**
- running with a `basic` license, I can see the `Schedule Reports` button in Discover. scheduled reporting is not available in `basic` even though single CSV exports is. We should hide the button
- same comment about `basic` license and `Schedule Report` button in Lens. We allow CSV exports through lens but should not allow scheduling reports
- export button for Dashboard correctly hidden for `basic` license 👍 

**Email notifications**

- email option correctly disabled when notifications are not properly configured 👍 
- email only allows setting `to` option. do we want to allow setting `cc` and `bcc` addresses?
- if I click `Schedule` without filling in an email address, I get a validation error in the `to` field and an error toast is shown. The schedule button is disabled and shows a loading spinner. Filling out the `to` field does not reenable the schedule button.

**Security or API keys turned off**

- when ES security is false or API keys are disabled, we show the `Schedule export` button but open a flyout that just shows an error. The `Schedule` button on the flyout is still enabled and an error toast shows up if you click it. Do we want to just show a disabled `Schedule export` button in the dropdown and skip opening the flyout altogether?

**Scheduling**

- seems to me that the `now` date is set when the flyout is opened, not when the request is made, so let's say you open the flyout at 8:00, mess around with the options and don't click the schedule button until 8:03. The schedule will be set to recur (daily/weekly/monthly) at 8:00.
- could we show the HH:MM time when scheduling? right now it says `Repeats every Wednesday`. Could we have it say `Repeats every Wednesday at 08:00`?

umbopepato committed:
Fix form validity check, flyout sizing, license check

umbopepato commented:
Thanks for taking a look Ying! I still haven't had time to update the description and add comments, 

**License**

> * running with a `basic` license, I can see the `Schedule Reports` button in Discover. scheduled reporting is not available in `basic` even though single CSV exports is. We should hide the button
> * same comment about `basic` license and `Schedule Report` button in Lens. We allow CSV exports through lens but should not allow scheduling reports
> * export button for Dashboard correctly hidden for `basic` license 👍

I thought the `prerequisiteCheck` was removed by the new share plugin extension point, it turns out it's not (I was just trying to add it in the wrong place)! I added back the license check, which according to the conversation on licensing you had with Tia on Slack seems like it's `license.type !== 'basic'`? If it's more nuanced let me know and I can update it.

**Email notifications**

> * email only allows setting `to` option. do we want to allow setting `cc` and `bcc` addresses?

I know the `notification` object of the schedule API has more options but the mockups only show a `To` field so I concentrated on that one. @tiamliu and @joana-cps probably wanted to start from a simple configuration and add the other recipient fields in the future?

> * if I click `Schedule` without filling in an email address, I get a validation error in the `to` field and an error toast is shown. The schedule button is disabled and shows a loading spinner. Filling out the `to` field does not reenable the schedule button.

My bad! I removed an intermediate component level last minute to remove a hack to adapt to the new share API and missed a `form.validate()` call. Fixed!

**Security or API keys turned off**

> * when ES security is false or API keys are disabled, we show the `Schedule export` button but open a flyout that just shows an error. The `Schedule` button on the flyout is still enabled and an error toast shows up if you click it. Do we want to just show a disabled `Schedule export` button in the dropdown and skip opening the flyout altogether?

This is a bit tricky since the health API call would need to be moved outside or React (and thus React Query) and in a check function that is not async... Let me see what I can do.

**Scheduling**

> * seems to me that the `now` date is set when the flyout is opened, not when the request is made, so let's say you open the flyout at 8:00, mess around with the options and don't click the schedule button until 8:03. The schedule will be set to recur (daily/weekly/monthly) at 8:00.

This is something we discussed in a meeting last Friday with @tiamliu. If I remember correctly the point was not confusing the user with a preview that doesn't correspond to the actual time used for scheduling (I have to pass a start date right away to the recurrence form, and all suggestions will be based on that date, but then the real schedule would be based on a different time - when executing the request - that the user doesn't have visibility on). I know it's not ideal and I'm open to all possibilities.

> * could we show the HH:MM time when scheduling? right now it says `Repeats every Wednesday`. Could we have it say `Repeats every Wednesday at 08:00`?

Sure, I'll extend the preview generator.

umbopepato committed:
Use loading state from rq instead of buggy form.isSubmitting

umbopepato committed:
Add time to recurrence form textual summary

umbopepato committed:
Attempt to fix functional Discover tests

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310108)
* Commit: 4b7a882a7b3ae3e1f7185693aacfb054bce6094c

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/310108#019783f0-2018-44ef-92c6-dceee8891c3b)

### History
* :green_heart: [Build #310054](https://buildkite.com/elastic/kibana-pull-request/builds/310054) succeeded 9daa2b8f97b06745747ec1b33187b880d9bfc180
* :broken_heart: [Build #309869](https://buildkite.com/elastic/kibana-pull-request/builds/309869) failed c4a41dd22c70f6e576d757d88db271efd23567f5
* :broken_heart: [Build #309807](https://buildkite.com/elastic/kibana-pull-request/builds/309807) failed e6d746b82cc27d3d1dee895c354c93a70853ef0f
* :broken_heart: [Build #309752](https://buildkite.com/elastic/kibana-pull-request/builds/309752) failed 4d63748d828cdedbd07f3124bc767a742d2113cd
* :broken_heart: [Build #303914](https://buildkite.com/elastic/kibana-pull-request/builds/303914) failed 0d3db10a6c4421a1bdf694b06ff55f172dbc3f51
* :broken_heart: [Build #303913](https://buildkite.com/elastic/kibana-pull-request/builds/303913) failed 4dd68c2f3692ed1c0bd5b9ae70e3a1cafab12d22

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310108","number":310108,"commit":"4b7a882a7b3ae3e1f7185693aacfb054bce6094c"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310054","number":310054,"commit":"9daa2b8f97b06745747ec1b33187b880d9bfc180"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309869","number":309869,"commit":"c4a41dd22c70f6e576d757d88db271efd23567f5"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309807","number":309807,"commit":"e6d746b82cc27d3d1dee895c354c93a70853ef0f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309752","number":309752,"commit":"4d63748d828cdedbd07f3124bc767a742d2113cd"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303914","number":303914,"commit":"0d3db10a6c4421a1bdf694b06ff55f172dbc3f51"}],"number":310108}
buildkite-pr-comment-->

umbopepato committed:
Add pdf print option toggle

umbopepato committed:
Add health check before registering button, fix type error

umbopepato committed:
Disable yearly frequency

umbopepato committed:
Fix Discover functional tests

umbopepato committed:
Revert test change causing type error

umbopepato committed:
Implement scheduled report transform util, update recurrence form

umbopepato committed:
Fix type error

umbopepato commented on src/platform/packages/private/kbn-reporting/public/index.ts:
```diff
@@ -26,10 +28,13 @@ import type { SharePluginStart } from '@kbn/share-plugin/public';
   docLinks: CoreStart['docLinks'];
   data: DataPublicPluginStart;
   share: SharePluginStart;
+  actions: ActionsPublicPluginSetup;
```
We need the actions plugin to get the `validateEmailAddresses` utility (not ideal, but in the future we could export that to a more generic place)

umbopepato commented on src/platform/packages/shared/response-ops/recurring-schedule-form/components/recurring_schedule_form_fields.tsx:
```diff
@@ -39,30 +40,26 @@ import { parseSchedule } from '../utils/parse_schedule';
 const UseField = getUseField({ component: Field });
-export const toMoment = (value: string): Moment => moment(value);
-export const toString = (value: Moment): string => value.toISOString();
+export const toMoment = (value?: string): Moment | undefined => (value ? moment(value) : undefined);
```
Unlike in MWs, here we have some nullable dates (i.e. `until`). Without this change the form would display `Invalid date` with empty dates

umbopepato committed:
Catch schedule report errors

umbopepato committed:
Fix last Dashboard and Discover functional tests

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310567)
* Commit: 9673dbffe0c566ded4f99c01a63b2c722210ab2a

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/310567#019788f6-9b38-437f-88e9-89b92520ec65)

### History
* :broken_heart: [Build #310512](https://buildkite.com/elastic/kibana-pull-request/builds/310512) failed 65dea18ba3fb69a9be2f26df32c7fb410afbb13f
* :broken_heart: [Build #310473](https://buildkite.com/elastic/kibana-pull-request/builds/310473) failed a492c73a15040ebe0cc3ca694d63b9c03fd1c073
* :broken_heart: [Build #310405](https://buildkite.com/elastic/kibana-pull-request/builds/310405) failed 364b8d6827f01c229d213135bbfc3a89bae092ef
* :broken_heart: [Build #310317](https://buildkite.com/elastic/kibana-pull-request/builds/310317) failed e9bceb9a920d2b3d21d00f1ae11332c20b70ad60
* :broken_heart: [Build #310289](https://buildkite.com/elastic/kibana-pull-request/builds/310289) failed 70897028b23c95cac79c57d34471991d82525d92

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310567","number":310567,"commit":"9673dbffe0c566ded4f99c01a63b2c722210ab2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310512","number":310512,"commit":"65dea18ba3fb69a9be2f26df32c7fb410afbb13f"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310473","number":310473,"commit":"a492c73a15040ebe0cc3ca694d63b9c03fd1c073"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310405","number":310405,"commit":"364b8d6827f01c229d213135bbfc3a89bae092ef"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310317","number":310317,"commit":"e9bceb9a920d2b3d21d00f1ae11332c20b70ad60"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310289","number":310289,"commit":"70897028b23c95cac79c57d34471991d82525d92"}],"number":310567}
buildkite-pr-comment-->

umbopepato committed:
Fix wrong page object method call

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310597)
* Commit: 4b87914e5ec4b0b0e75917c05be2e38140169403
* [Cloud Deployment](https://kibana-pr-222135.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #310567](https://buildkite.com/elastic/kibana-pull-request/builds/310567) failed 9673dbffe0c566ded4f99c01a63b2c722210ab2a
* :broken_heart: [Build #310512](https://buildkite.com/elastic/kibana-pull-request/builds/310512) failed 65dea18ba3fb69a9be2f26df32c7fb410afbb13f
* :broken_heart: [Build #310473](https://buildkite.com/elastic/kibana-pull-request/builds/310473) failed a492c73a15040ebe0cc3ca694d63b9c03fd1c073
* :broken_heart: [Build #310405](https://buildkite.com/elastic/kibana-pull-request/builds/310405) failed 364b8d6827f01c229d213135bbfc3a89bae092ef
* :broken_heart: [Build #310317](https://buildkite.com/elastic/kibana-pull-request/builds/310317) failed e9bceb9a920d2b3d21d00f1ae11332c20b70ad60

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310597","number":310597,"commit":"4b87914e5ec4b0b0e75917c05be2e38140169403"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310567","number":310567,"commit":"9673dbffe0c566ded4f99c01a63b2c722210ab2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310512","number":310512,"commit":"65dea18ba3fb69a9be2f26df32c7fb410afbb13f"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310473","number":310473,"commit":"a492c73a15040ebe0cc3ca694d63b9c03fd1c073"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310405","number":310405,"commit":"364b8d6827f01c229d213135bbfc3a89bae092ef"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310317","number":310317,"commit":"e9bceb9a920d2b3d21d00f1ae11332c20b70ad60"}],"number":310597}
buildkite-pr-comment-->

umbopepato committed:
More functional test fixes

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310622)
* Commit: d7b861ccfdd0749c882f8f94edec91c9d491ffe2
* [Cloud Deployment](https://kibana-pr-222135.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #310597](https://buildkite.com/elastic/kibana-pull-request/builds/310597) failed 4b87914e5ec4b0b0e75917c05be2e38140169403
* :broken_heart: [Build #310567](https://buildkite.com/elastic/kibana-pull-request/builds/310567) failed 9673dbffe0c566ded4f99c01a63b2c722210ab2a
* :broken_heart: [Build #310512](https://buildkite.com/elastic/kibana-pull-request/builds/310512) failed 65dea18ba3fb69a9be2f26df32c7fb410afbb13f
* :broken_heart: [Build #310473](https://buildkite.com/elastic/kibana-pull-request/builds/310473) failed a492c73a15040ebe0cc3ca694d63b9c03fd1c073
* :broken_heart: [Build #310405](https://buildkite.com/elastic/kibana-pull-request/builds/310405) failed 364b8d6827f01c229d213135bbfc3a89bae092ef

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310622","number":310622,"commit":"d7b861ccfdd0749c882f8f94edec91c9d491ffe2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310597","number":310597,"commit":"4b87914e5ec4b0b0e75917c05be2e38140169403"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310567","number":310567,"commit":"9673dbffe0c566ded4f99c01a63b2c722210ab2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310512","number":310512,"commit":"65dea18ba3fb69a9be2f26df32c7fb410afbb13f"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310473","number":310473,"commit":"a492c73a15040ebe0cc3ca694d63b9c03fd1c073"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310405","number":310405,"commit":"364b8d6827f01c229d213135bbfc3a89bae092ef"}],"number":310622}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310638)
* Commit: 7ff7bdba85defbbd66d7bfd62acb89b980537122
* [Cloud Deployment](https://kibana-pr-222135.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #310622](https://buildkite.com/elastic/kibana-pull-request/builds/310622) failed d7b861ccfdd0749c882f8f94edec91c9d491ffe2
* :broken_heart: [Build #310597](https://buildkite.com/elastic/kibana-pull-request/builds/310597) failed 4b87914e5ec4b0b0e75917c05be2e38140169403
* :broken_heart: [Build #310567](https://buildkite.com/elastic/kibana-pull-request/builds/310567) failed 9673dbffe0c566ded4f99c01a63b2c722210ab2a
* :broken_heart: [Build #310512](https://buildkite.com/elastic/kibana-pull-request/builds/310512) failed 65dea18ba3fb69a9be2f26df32c7fb410afbb13f
* :broken_heart: [Build #310473](https://buildkite.com/elastic/kibana-pull-request/builds/310473) failed a492c73a15040ebe0cc3ca694d63b9c03fd1c073

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310638","number":310638,"commit":"7ff7bdba85defbbd66d7bfd62acb89b980537122"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310622","number":310622,"commit":"d7b861ccfdd0749c882f8f94edec91c9d491ffe2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310597","number":310597,"commit":"4b87914e5ec4b0b0e75917c05be2e38140169403"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310567","number":310567,"commit":"9673dbffe0c566ded4f99c01a63b2c722210ab2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310512","number":310512,"commit":"65dea18ba3fb69a9be2f26df32c7fb410afbb13f"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310473","number":310473,"commit":"a492c73a15040ebe0cc3ca694d63b9c03fd1c073"}],"number":310638}
buildkite-pr-comment-->

jcger commented on x-pack/platform/plugins/private/reporting/public/management/components/responsive_form_group.tsx:
```diff
@@ -0,0 +1,34 @@
+ * A collapsible version of EuiDescribedFormGroup. Use the `narrow` prop
+ * to obtain a vertical layout suitable for smaller forms
+ */
+export const ResponsiveFormGroup = ({
```
FWIW I did it this way https://github.com/elastic/kibana/blob/b759ebba3da4fa3f34fa914f99017c315a3294af/src/platform/packages/shared/response-ops/alerts-delete/components/descriptive_form_group.tsx#L66, which should be responsive by default

umbopepato replied:
Ah that's nice! We could propose Eui to integrate your solution in the original one 🙂

umbopepato committed:
Even more functional test fixes

umbopepato commented on x-pack/platform/plugins/private/reporting/public/plugin.ts:
```diff
@@ -234,6 +244,22 @@ export class ReportingPublicPlugin
       );
     }
 
+    import('./management/integrations/scheduled_report_share_integration').then(
```
When imported statically it broke the page load bundle size limit for reporting, see
https://buildkite.com/elastic/kibana-pull-request/builds/309752/summary/annotations?jid=01978228-643a-4d7d-810e-4cd9e2596513

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310779)
* Commit: ff6ff75af604d3029085966afbbc322596fecaec
* [Cloud Deployment](https://kibana-pr-222135.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #310638](https://buildkite.com/elastic/kibana-pull-request/builds/310638) failed 7ff7bdba85defbbd66d7bfd62acb89b980537122
* :broken_heart: [Build #310622](https://buildkite.com/elastic/kibana-pull-request/builds/310622) failed d7b861ccfdd0749c882f8f94edec91c9d491ffe2
* :broken_heart: [Build #310597](https://buildkite.com/elastic/kibana-pull-request/builds/310597) failed 4b87914e5ec4b0b0e75917c05be2e38140169403
* :broken_heart: [Build #310567](https://buildkite.com/elastic/kibana-pull-request/builds/310567) failed 9673dbffe0c566ded4f99c01a63b2c722210ab2a
* :broken_heart: [Build #310512](https://buildkite.com/elastic/kibana-pull-request/builds/310512) failed 65dea18ba3fb69a9be2f26df32c7fb410afbb13f

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310779","number":310779,"commit":"ff6ff75af604d3029085966afbbc322596fecaec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310638","number":310638,"commit":"7ff7bdba85defbbd66d7bfd62acb89b980537122"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310622","number":310622,"commit":"d7b861ccfdd0749c882f8f94edec91c9d491ffe2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310597","number":310597,"commit":"4b87914e5ec4b0b0e75917c05be2e38140169403"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310567","number":310567,"commit":"9673dbffe0c566ded4f99c01a63b2c722210ab2a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310512","number":310512,"commit":"65dea18ba3fb69a9be2f26df32c7fb410afbb13f"}],"number":310779}
buildkite-pr-comment-->

joana-cps reviewed (commented):
<img width="1646" alt="Screenshot 2025-06-20 at 13 08 50" src="https://github.com/user-attachments/assets/f0ba0622-34b4-4bc2-97a0-d3900045607c" />

The `Reporting page` link should point to the reporting page with the exports tab selected. Currently, it is pointing to the schedules tab, and we are not listing the reports there. 

Can you also make it open in a new browser tab?

joana-cps reviewed (commented):
<img width="1648" alt="Screenshot 2025-06-20 at 13 03 51" src="https://github.com/user-attachments/assets/f2e736dd-87c0-4ee3-b813-0a046937c1ec" />

It feels weird to see the `Exports` sections in the `View schedule information` read-only flyout, especially when we display the callout about the email connector. 
Should we hide this section altogether if 'Send by email' is off?

joana-cps reviewed (commented):
<img width="495" alt="Screenshot 2025-06-20 at 13 21 12" src="https://github.com/user-attachments/assets/01142bb1-e413-4e38-9263-e39abd4beec1" />

The `Daily` behaviour is strange. If we display the days of the week, they should all be selected by default when the user selects `Daily`. Otherwise, even though I select `Daily` only friday is selected and a report is exported as "Repeats every Friday at xxx".

joana-cps reviewed (commented):
<img width="239" alt="Screenshot 2025-06-20 at 15 50 51" src="https://github.com/user-attachments/assets/19e84575-2926-476e-bb9f-9f02118922aa" />
<img width="244" alt="Screenshot 2025-06-20 at 15 50 42" src="https://github.com/user-attachments/assets/54e203fd-dedb-438e-b4cd-8a61800095b2" />

Inconsistent icon for CSV export. Since the `documents` icon will be deprecated, let's use `tableDensityExpanded` for CSV and `document` for PDF.

umbopepato commented:
> <img alt="Screenshot 2025-06-20 at 13 08 50" width="1646" src="https://private-user-images.githubusercontent.com/169058851/457336081-f0ba0622-34b4-4bc2-97a0-d3900045607c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA0Mjk4MDgsIm5iZiI6MTc1MDQyOTUwOCwicGF0aCI6Ii8xNjkwNTg4NTEvNDU3MzM2MDgxLWYwYmEwNjIyLTM0YjQtNGJjMi05N2EwLWQzOTAwMDQ1NjA3Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYyMFQxNDI1MDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYzA3ZTA4ZDlkNGFkMTk1YmRmYjg3OTgyMzZlNTQ3ZTY0YmEwNWQ3OWI3MjAwNjAzYjc0MmQ2NWE0YzVlYjhhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.1DoChhD0xV_qENA_b6LogUvjeysHvNgQ3dYYm4HGRaQ">
> The `Reporting page` link should point to the reporting page with the exports tab selected. Currently, it is pointing to the schedules tab, and we are not listing the reports there.
> 
> Can you also make it open in a new browser tab?

This is because in this PR I don't have the other tab, this can be done when integrating the flyout with the list page

umbopepato commented:
> <img alt="Screenshot 2025-06-20 at 13 21 12" width="495" src="https://private-user-images.githubusercontent.com/169058851/457339478-01142bb1-e413-4e38-9263-e39abd4beec1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA0Mjk4MDgsIm5iZiI6MTc1MDQyOTUwOCwicGF0aCI6Ii8xNjkwNTg4NTEvNDU3MzM5NDc4LTAxMTQyYmIxLWU0MTMtNGUzOC05MjYzLWUzOWFiZDRiZWVjMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYyMFQxNDI1MDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNzU0MmNmNzQ0YjVmMmNkY2VkYWRhODE4MDQzMjQ5NjI3ZWViZGU4NjBlZTkwZTRiYTA3YTUyYjJjMWFlZDc3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QVLh99xVI3wbc_DnTOq7plHZDSH7OeSXqzNzD2Y7OuI">
> The `Daily` behaviour is strange. If we display the days of the week, they should all be selected by default when the user selects `Daily`. Otherwise, even though I select `Daily` only friday is selected and a report is exported as "Repeats every Friday at xxx".

Good point! That part of the form is also shared with MWs though so let's maybe track this as a separate improvement?

ymao1 commented on x-pack/platform/plugins/private/reporting/public/management/integrations/scheduled_report_share_integration.tsx:
```diff
@@ -0,0 +1,71 @@
+      if (!license) {
+        return false;
+      }
+      return license.type !== 'basic';
```
We added an allowlist for licenses here: https://github.com/elastic/kibana/pull/221028/commits/f5f9d9daedcd18447b6a02335a53631a44413788 Not sure if you can reuse.

umbopepato replied:
I couldn't import it from there since it's marked as server code so I moved it to the `common` folder, hope it's fine 🙂

umbopepato committed:
Filter share items by ones that support scheduling

umbopepato committed:
Change csv export icon type

umbopepato committed:
Log schedule errors to console

umbopepato committed:
Check license against allowlist

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/notice'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311013)
* Commit: 03d14c943dafcb436edd2bf2f8cbba950fbd7fde

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/311013#01978e18-694d-47a5-8297-4ce425d87d7f)

### History
* :green_heart: [Build #310779](https://buildkite.com/elastic/kibana-pull-request/builds/310779) succeeded ff6ff75af604d3029085966afbbc322596fecaec
* :broken_heart: [Build #310638](https://buildkite.com/elastic/kibana-pull-request/builds/310638) failed 7ff7bdba85defbbd66d7bfd62acb89b980537122
* :broken_heart: [Build #310622](https://buildkite.com/elastic/kibana-pull-request/builds/310622) failed d7b861ccfdd0749c882f8f94edec91c9d491ffe2
* :broken_heart: [Build #310597](https://buildkite.com/elastic/kibana-pull-request/builds/310597) failed 4b87914e5ec4b0b0e75917c05be2e38140169403
* :broken_heart: [Build #310567](https://buildkite.com/elastic/kibana-pull-request/builds/310567) failed 9673dbffe0c566ded4f99c01a63b2c722210ab2a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311013","number":311013,"commit":"03d14c943dafcb436edd2bf2f8cbba950fbd7fde"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310779","number":310779,"commit":"ff6ff75af604d3029085966afbbc322596fecaec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310638","number":310638,"commit":"7ff7bdba85defbbd66d7bfd62acb89b980537122"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310622","number":310622,"commit":"d7b861ccfdd0749c882f8f94edec91c9d491ffe2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310597","number":310597,"commit":"4b87914e5ec4b0b0e75917c05be2e38140169403"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310567","number":310567,"commit":"9673dbffe0c566ded4f99c01a63b2c722210ab2a"}],"number":311013}
buildkite-pr-comment-->

umbopepato committed:
Fix circular TS proj dependency

eokoneyo commented:
> <img alt="Screenshot 2025-06-20 at 15 50 51" width="239" src="https://private-user-images.githubusercontent.com/169058851/457398624-19e84575-2926-476e-bb9f-9f02118922aa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA2NjI0MjUsIm5iZiI6MTc1MDY2MjEyNSwicGF0aCI6Ii8xNjkwNTg4NTEvNDU3Mzk4NjI0LTE5ZTg0NTc1LTI5MjYtNDc2ZS1iYjlmLTlmMDIxMTg5MjJhYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYyM1QwNzAyMDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00M2MzZGFjMjk2Y2Q0ZTZiM2QwZjRkYTA4ODk0ZWI1NGYyZGJmOThkZTQzOWE2YTRkODk3MDZkYzdlNTlkOTE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Z5IxPZGKg05IPvqVNIS3OH68JIUYWkJunily6lJCz60"> <img alt="Screenshot 2025-06-20 at 15 50 42" width="244" src="https://private-user-images.githubusercontent.com/169058851/457398631-54e203fd-dedb-438e-b4cd-8a61800095b2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA2NjI0MjUsIm5iZiI6MTc1MDY2MjEyNSwicGF0aCI6Ii8xNjkwNTg4NTEvNDU3Mzk4NjMxLTU0ZTIwM2ZkLWRlZGItNDM4ZS1iNGNkLThhNjE4MDAwOTViMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYyM1QwNzAyMDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZGZhMTY4MmQxM2RkMDQ1YmVmNTAwYWU1ODdiNzhkNWIxYTRlMjI3YWEyYmMwOThhNTJmZTcxOWU1MDg1ODIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.r8HTwO9nUVScYnHYGMmo0EqtbCxPO3hruWOmOGXNUZM">
> Inconsistent icon for CSV export. Since the `documents` icon will be deprecated, let's use `tableDensityExpanded` for CSV and `document` for PDF.

I have a PR for this [here](https://github.com/elastic/kibana/pull/224752)

umbopepato committed:
Combine import and export of useShareTypeContext into a single statement

Co-authored-by: Eyo O. Eyo <7893459+eokoneyo@users.noreply.github.com>

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/yarn_deduplicate'

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311264)
* Commit: bc6aa0c7f80dae401dd68b3b0c5576a43db89de3

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/311264#01979bae-71f9-45b4-9633-0d04d5f7415c)

### History
* :broken_heart: [Build #311013](https://buildkite.com/elastic/kibana-pull-request/builds/311013) failed 03d14c943dafcb436edd2bf2f8cbba950fbd7fde
* :green_heart: [Build #310779](https://buildkite.com/elastic/kibana-pull-request/builds/310779) succeeded ff6ff75af604d3029085966afbbc322596fecaec
* :broken_heart: [Build #310638](https://buildkite.com/elastic/kibana-pull-request/builds/310638) failed 7ff7bdba85defbbd66d7bfd62acb89b980537122
* :broken_heart: [Build #310622](https://buildkite.com/elastic/kibana-pull-request/builds/310622) failed d7b861ccfdd0749c882f8f94edec91c9d491ffe2
* :broken_heart: [Build #310597](https://buildkite.com/elastic/kibana-pull-request/builds/310597) failed 4b87914e5ec4b0b0e75917c05be2e38140169403

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311264","number":311264,"commit":"bc6aa0c7f80dae401dd68b3b0c5576a43db89de3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311013","number":311013,"commit":"03d14c943dafcb436edd2bf2f8cbba950fbd7fde"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310779","number":310779,"commit":"ff6ff75af604d3029085966afbbc322596fecaec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310638","number":310638,"commit":"7ff7bdba85defbbd66d7bfd62acb89b980537122"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310622","number":310622,"commit":"d7b861ccfdd0749c882f8f94edec91c9d491ffe2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310597","number":310597,"commit":"4b87914e5ec4b0b0e75917c05be2e38140169403"}],"number":311264}
buildkite-pr-comment-->

umbopepato committed:
Fix type error

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311278)
* Commit: bc68511cf15ef832cc9fe9b5b0110b72e94baa1d
* [Cloud Deployment](https://kibana-pr-222135.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #311264](https://buildkite.com/elastic/kibana-pull-request/builds/311264) failed bc6aa0c7f80dae401dd68b3b0c5576a43db89de3
* :broken_heart: [Build #311013](https://buildkite.com/elastic/kibana-pull-request/builds/311013) failed 03d14c943dafcb436edd2bf2f8cbba950fbd7fde
* :green_heart: [Build #310779](https://buildkite.com/elastic/kibana-pull-request/builds/310779) succeeded ff6ff75af604d3029085966afbbc322596fecaec
* :broken_heart: [Build #310638](https://buildkite.com/elastic/kibana-pull-request/builds/310638) failed 7ff7bdba85defbbd66d7bfd62acb89b980537122
* :broken_heart: [Build #310622](https://buildkite.com/elastic/kibana-pull-request/builds/310622) failed d7b861ccfdd0749c882f8f94edec91c9d491ffe2
* :broken_heart: [Build #310597](https://buildkite.com/elastic/kibana-pull-request/builds/310597) failed 4b87914e5ec4b0b0e75917c05be2e38140169403

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311278","number":311278,"commit":"bc68511cf15ef832cc9fe9b5b0110b72e94baa1d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311264","number":311264,"commit":"bc6aa0c7f80dae401dd68b3b0c5576a43db89de3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311013","number":311013,"commit":"03d14c943dafcb436edd2bf2f8cbba950fbd7fde"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310779","number":310779,"commit":"ff6ff75af604d3029085966afbbc322596fecaec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310638","number":310638,"commit":"7ff7bdba85defbbd66d7bfd62acb89b980537122"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310622","number":310622,"commit":"d7b861ccfdd0749c882f8f94edec91c9d491ffe2"}],"number":311278}
buildkite-pr-comment-->
